app/main.py

```diff
@@ -1,11 +1,14 @@
 import spade
+
 from agents.JobOfferManagerAgent import JobOfferManagerAgent
+from agents.ApplicationAnalyzerAgent import ApplicationAnalyzerAgent
 from dataaccess.model.JobOffer import JobOffer
 from modules.JobOfferModule import JobOfferModule
 from utils.log_config import LogConfig
 
 from app.utils.configuration import MASConfiguration
 
+ANALYZER_INSTANCES = 2
```

Comment:
możnaby przenieść do config.json

Reply:
zrobiłem to na swoim branchu. Wiec tutaj tego nie ruszac